cucumber.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+#Run server in background
+http-server ./test/demo-app -p 9000 > /dev/null &
```

Comment:
Shouldn't it be `./node_modules/.bin/http-server ./test/demo-app -p 9000 > /dev/null &` ?

Reply:
Agree. Fixed.